addons/popochiu/engine/objects/graphic_interface/sound_settings/sound_settings.tscn

```diff
@@ -0,0 +1,100 @@
+[ext_resource type="Script" path="res://addons/popochiu/engine/objects/graphic_interface/sound_settings/sound_settings.gd" id="2_xvxxy"]
+
+[sub_resource type="StyleBoxFlat" id="StyleBoxFlat_st5xf"]
+bg_color = Color(0, 0, 0, 0.498039)
+
+[sub_resource type="StyleBoxFlat" id="StyleBoxFlat_ych3d"]
+content_margin_left = 8.0
+content_margin_top = 4.0
+content_margin_right = 8.0
+content_margin_bottom = 4.0
+bg_color = Color(0.180392, 0.172549, 0.607843, 1)
+shadow_color = Color(0, 0, 0, 1)
+shadow_size = 4
+shadow_offset = Vector2(0, 2)
+
+[node name="SoundSettings" type="PanelContainer"]
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+theme = ExtResource("1_k50u1")
+theme_override_styles/panel = SubResource("StyleBoxFlat_st5xf")
+script = ExtResource("2_xvxxy")
+
+[node name="SoundSettingsDialog" type="PanelContainer" parent="."]
+layout_mode = 2
+size_flags_horizontal = 4
+size_flags_vertical = 4
+theme_override_styles/panel = SubResource("StyleBoxFlat_ych3d")
+
+[node name="VBoxContainer" type="VBoxContainer" parent="SoundSettingsDialog"]
+custom_minimum_size = Vector2(200, 0)
+layout_mode = 2
+
+[node name="Title" type="Label" parent="SoundSettingsDialog/VBoxContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+size_flags_horizontal = 4
+text = "Sound Settings"
+
+[node name="HBoxContainer" type="HBoxContainer" parent="SoundSettingsDialog/VBoxContainer"]
+layout_mode = 2
+
+[node name="Channels" type="VBoxContainer" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer"]
+unique_name_in_owner = true
+custom_minimum_size = Vector2(50, 50)
+layout_mode = 2
+alignment = 1
+
+[node name="MasterBtn" type="Label" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer/Channels"]
+layout_mode = 2
+text = "Master"
+
+[node name="MusicBtn" type="Label" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer/Channels"]
+layout_mode = 2
+text = "Music"
+
+[node name="SFXBtn" type="Label" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer/Channels"]
+layout_mode = 2
+text = "SFX"
+vertical_alignment = 1
+
+[node name="VBoxContainer" type="VBoxContainer" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer"]
+custom_minimum_size = Vector2(145, 0)
+layout_mode = 2
+size_flags_vertical = 4
+
+[node name="MasterSlider" type="HSlider" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer/VBoxContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+min_value = -70.0
+max_value = 0.0
+
+[node name="MusicSlider" type="HSlider" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer/VBoxContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+min_value = -80.0
+max_value = 0.0
+
+[node name="SFXSlider" type="HSlider" parent="SoundSettingsDialog/VBoxContainer/HBoxContainer/VBoxContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+min_value = -65.0
+max_value = 0.0
```

Comment:
These should be created during runtime (except for Master) if we decide to leave developers to define their own buses.